analyses/cell-types-annotation/README.md

```diff
@@ -0,0 +1,65 @@
+# Pipeline for annotating cell types for sc-/sn-RNA-Seq Analysis in 10X Genomics data
+
+## Usage
+
+To run the script in this module from the command line sequentially, use:
+
+```
+bash run-cell-types-annotation.sh
```

Comment:
I noticed there are no instructions in the README for running this with an `lsf` job, but the module has the scripts to run it with `lsf`. Should those directions be added to the README?

Reply:
Yes, we shoudl add these. Fixed here 07844f073f294e9ce68db965b928d0b7fa142f85.